resources/lang/en/campaigns.php

```diff
@@ -417,13 +417,20 @@
         'connections'       => [
             'explorer'  => 'Relations explorer (if available, for boosted campaigns)',
             'list'      => 'List interface',
+            'default'   => 'Default',
```

Comment:
Why not reuse the existing translations, so that our translators don't need to work twice? :)